packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -72,7 +69,7 @@ export function EntitiesSavedStatesExtensible( {
 	onSave = identity,
 	saveEnabled: saveEnabledProp = undefined,
 	saveLabel = __( 'Save' ),
-	renderDialog = undefined,
+	renderDialog = false,
```

Comment:
Why change to `false`? Did it affect something by being `undefined`? I mean we could even remove the ` = undefined` part, no?

Reply:
Yes I considered to remove the ` = undefined`. I thought it's more explicit this way as it's a 'flag'. No strong opinions though.

Reply:
I think it's fine to remove.